research/format.markdown

```diff
@@ -0,0 +1,97 @@
+|-------------|------------------|----------------------------------------------------------------------------------------------------------------------|
+| Added       | `uri-reference`    | This attribute applies to string instances. A string instance is valid against this attribute if it is a valid URI Reference (either a URI or a relative-reference), according to [RFC3986](https://www.rfc-editor.org/info/rfc3986) |
+| Added       | `uri-template`     | This attribute applies to string instances. A string instance is valid against this attribute if it is a valid URI Template (of any level), according to [RFC6570](https://www.rfc-editor.org/info/rfc6570)                          |
+| Added       | `json-pointer`     | This attribute applies to string instances. A string instance is valid against this attribute if it is a valid JSON Pointer, according to [RFC6901](https://www.rfc-editor.org/info/rfc6901)                                            |
+
+## Draft 6 to Draft 7
+
+References: [Draft 6](https://json-schema.org/draft-06/draft-wright-json-schema-validation-01#rfc.section.8), [Draft 7](https://json-schema.org/draft-07/draft-handrews-json-schema-validation-01#rfc.section.7)
+
+| Change Type | Format Attribute           | Description                                                                                                          |
+|-------------|----------------------------|----------------------------------------------------------------------------------------------------------------------|
+| Added       | `date`                       | A string instance is valid against this attribute if it is a valid representation according to the `full-date` production                                |
+| Added       | `time`                       | A string instance is valid against this attribute if it is a valid representation according to the `full-time` production                                |
+|             |                            | Implementations MAY support additional attributes using the other production names defined in that section. If `full-date` or `full-time` are implemented, the corresponding short form (`date` or `time` respectively) MUST be implemented and MUST behave identically. Implementations SHOULD NOT define extension attributes with any name matching an [RFC 3339](https://www.rfc-editor.org/info/rfc3339) production unless it validates according to the rules of that production |
+| Added       | `idn-email`                  | As defined by RFC 6531 [RFC6531](https://www.rfc-editor.org/info/rfc6531) |
+| | |idn-email is a superset of email              |
+| Added       | `hostname`                   | As defined by RFC 1034, section 3.1 [RFC1034](https://www.rfc-editor.org/info/rfc1034), including host names produced using the Punycode algorithm specified in RFC 5891, section 4.4 [RFC5891](https://www.rfc-editor.org/info/rfc5891) |
```

Comment:
When was `host-name` (which you referred to above) deleted? I don't see it in Draft 7 anymore, and no "deleted" entry in previous parts of this document either. Can you double-check we are not missing any deletes in all the drafts this document is covering?

Reply:
Yeah! let me recheck. I may have missed it.